backend/src/script.ts

```diff
@@ -32,4 +34,37 @@ const formatReview = (data: any): Review => ({
   photos: data.photos,
 });
 
+const formatLandlord = ({
+  id,
+  name,
+  contact,
+  avgRating,
+  photos,
+  reviews,
+  properties,
+}: // eslint-disable-next-line @typescript-eslint/no-explicit-any
+any): LandlordWithId => ({
+  id: id.toString(),
+  name,
+  contact,
+  avgRating: avgRating.length > 0 ? Number.parseFloat(avgRating) : 0,
+  photos: photos.length > 0 ? photos : [],
```

Comment:
If `photos` is a non-empty string, you would be writing a string instead of a list into the db, which would mess things up. This applies to the other properties in the json with a string as the value.